imagetool/src/main/resources/ImageTool.properties

```diff
@@ -32,7 +32,7 @@ IMG-0030=Additional build command file does not exist: {0}
 IMG-0031=No credentials provided. Cannot determine latestPSU
 IMG-0032=Failed to find latest PSU for {0}, version {1}
 IMG-0033=No credentials provided, skipping validation of patches
-IMG-0034=There are multiple patches found with id {0}, please specify the format as one of the following for the release you want
+IMG-0034=Patch ID {0} has multiple versions, please retry the command using one of the available patch versions: {1}
```

Comment:
Should it say to retry with the latest version of the patch?

Reply:
this message applies to any patch, not just OPatch patch.  So, its not always the "latest".  This is part of what needs to be changed in the permanent fix.